src/DynamoCore/DynamoCore.csproj

```diff
@@ -305,6 +305,10 @@ limitations under the License.
     <AppDesigner Include="Properties\" />
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\Engine\GraphLayout\GraphLayout.csproj">
```

Comment:
I assume, GraphLayout project doesn't depend on WPF or any windows specific libraries, otherwise adding a dependency on DynamoCore will break on other platforms.